extensions/amp-carousel/0.2/test-e2e/test-carousel.js

```diff
@@ -21,32 +21,27 @@ import {
 } from './helpers';
 
 describes.endtoend('AMP carousel', {
+  fixture: 'http://localhost:8000/test/manual/amp-carousel-0-2/basic.amp.html',
+  experiments: ['amp-carousel-v2'],
+  environment: 'shadow-demo',
```

Comment:
Is the plan that if you want to test under multiple environments, you create multiple describes sharing the same test runner function?

Reply:
Yes, or a `describes.repeated`.

Reply:
IMO one of the most important features for e2e testing is to automatically run a given test on all environments by default. Happy to chat about what this would require.

/cc @rsimha

Reply:
I'd imagine we can use a `describes.repeated` block to iterate across the different types of `Environment`s in [`AmpEnvironments`](https://github.com/ampproject/amphtml/blob/993e2a9786a88865a3aa4ec1a07ec89f040155e2/build-system/tasks/e2e/amp-driver.js#L25-L60), just like this code goes over the variants.

https://github.com/ampproject/amphtml/blob/ce617b724d9efdc1951ad644c6cc2fb6c4a1f51c/extensions/amp-mustache/0.1/test/test-amp-mustache.js#L22-L72

/cc @alabiaga

Reply:
@rsimha that's what I was imagining as well. This would be the easiest implementation for now. We can refactor later to allow passing an array and have the describes.e2e handle the repetition, if that is ok with everyone

Reply:
SGTM. I like this because it allows us to define a pattern in once place and reuse it across several tests.

/cc @estherkim